dev/io.openliberty.microprofile.reactive.messaging.internal_fat/fat/src/io/openliberty/microprofile/reactive/messaging/fat/kafka/nack/KafkaNackTest.java

```diff
@@ -97,10 +98,11 @@ public static void setup() throws Exception {
     public static void teardown() throws Exception {
         server.stopServer("CWMRX1003E.*nack-test-channel", // CWMRX1003E: An error occurred when sending a message to the Kafka broker. The error is: org.apache.kafka.common.errors.TimeoutException: Topic nack-test-channel not present in metadata after 5000 ms.
                           "CWMRX1011E.*KafkaNackTestException");
+        KafkaUtils.deleteKafkaTopics(KafkaTests.getAdminClient());
     }
```

Comment:
Do we usually do the deletion in a try-finally around `stopServer()`?

It's common for `stopServer` to throw an exception if the test fails or some infrastructure breaks which might cause knock-on failures here?